app/controllers/declaration/RoutingCountriesController.scala

```diff
@@ -45,9 +45,7 @@ class RoutingCountriesController @Inject()(
     extends FrontendController(mcc) with I18nSupport with ModelCacheable {
 
   def displayRoutingQuestion(mode: Mode, fastForward: Boolean): Action[AnyContent] = (authenticate andThen journeyType) { implicit request =>
-    val containsRoutingCountries: Boolean = request.cacheModel.locations.routingCountries.nonEmpty
-
-    if (fastForward && containsRoutingCountries) {
+    if (fastForward && request.cacheModel.containRoutingCountries) {
```

Comment:
maybe adding () to `containRoutingCountries` would increase readbility - in first moment i assume that is one of declaration field